### ISSUE TITLE
The value in the custom filtering component disappears as you type

### ISSUE THREAD

**Issue by AleksiupasSameSystem** [ag-grid/ag-grid#3403]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => see 'Providing a Reproducible Scenario'
[] feature request => do not use Github for feature requests, see 'Customers of ag-Grid'
[] support request => see 'Requesting Community Support'
```

**Hello**,

The custom filtering component is completely rerendered after calling the `filterChangedCallback()` function when `reactNext={true}`and at least one colDef has `cellRendererFramework`.

The problem is that the value in the custom filtering component disappears as you type.

good
[Example: Filtering using React Components](https://www.ag-grid.com/javascript-grid-filter-component/#example-filtering-using-react-components)

bad
Try filtering the filter component column.
[Example: Filtering using React Components(reactNext+cellRendererFramework)](https://next.plnkr.co/edit/bzNGVEPGnerSlwc9)

**Comment by seanlandsman** [comment]
Hi

Thanks for reporting this - this has been fixed and will be available in our next release.

**Comment by seanlandsman** [comment]
@AleksiupasSameSystem I've released ag-grid-react@22.0.0-beta.0 to allow users to test the fix - let me know if this works for you.

**Comment by AleksiupasSameSystem** [comment]
Now it seems to work, thank you.
